### ISSUE TITLE
Build Failed on Xcode 8.3.3 and 9.0.1 on High Sierra

### ISSUE THREAD

**Issue by jeanfleblanc** [MailCore/mailcore2#1675]
I get the following error when I try to build mailcore osx version with Xcode 8.3.3 and 9.0.1:

~~~~
ld: library not found for -lssl
clang: error: linker command failed with exit code 1 (use -v to see invocation)
~~~~

The mailcore iOS version works fine.

**Comment by dinhviethoa** [comment]
What are you building / How are you integrating with your app?

**Comment by jeanfleblanc** [comment]
Trying to build mailcore2 as a carthage dependency.

**Comment by dinhviethoa** [comment]
Did you add manually the -lssl flag?
Could you show a more complete log of the error?

**Comment by jeanfleblanc** [comment]
I put this in the carthage file:

~~~~
github "MailCore/mailcore2" 
~~~~

And run:

~~~~
carthage update
~~~~

Carthage downloaded MailCore2 version  0.6.2 and tried to build it, and then I got the error:

~~~~
ld: library not found for -lssl
clang: error: linker command failed with exit code 1 (use -v to see invocation)
~~~~

If I remove -lssl and -lcrypto manually from the project file, it builds successfully.

**Comment by dinhviethoa** [comment]
Could you try:
```
github "MailCore/mailcore2" HEAD
```

**Comment by jeanfleblanc** [comment]
It's working!

**Comment by ikemuc** [comment]
Hi,

I'm again back at trying to build mailcore2 on Mac OS 10.13 using carthage (migrating the project from Cocoapods to Carthage) and I am fighting against the same problem.

When I create the plain Cartfile with
     github "MailCore/mailcore2"
and do a "carthage update", I get the following error

*** Building scheme "mailcore osx" in mailcore2.xcodeproj
Build Failed
	Task failed with exit code 65:

And in the build log:

ld: library not found for -lssl
clang: error: linker command failed with exit code 1 (use -v to see invocation)

I tried
     github "MailCore/mailcore2" HEAD
also, same result.

I'm using Xcode 9.3 and 9.4 beta 2 on MacOS 10.13.4. I removed the whole Xcode installation already and reinstalled everything. From other posts on the web, I learned, that OpenSSL is not part of MacOS any more, which seems to be the problem. I also tried installing OpenSSL via homebrew, but this didn't work also...

Does anyone have a hint for me??? Who has built mailcore2 on MacOS 10.13 successfully??? Help desperately need...
